plugin/ochttp/example_test.go

```diff
@@ -20,9 +20,31 @@ import (
 
 	"go.opencensus.io/plugin/ochttp"
 	"go.opencensus.io/plugin/ochttp/propagation/google"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
 )
 
 func ExampleTransport() {
+
+	err := view.Subscribe(
+
+		// Subscribe to a few default views, with renaming
```

Comment:
I don't think we should ever have an example of WithName at ochttp, this is such as edge-case feature.

An example can be in the stats package.

Reply:
To be honest, I think it will not be an edge-case feature. Our view names for HTTP and gRPC are so long I would almost certainly rename them if I were an using OC in an application. I feel like we haven't got this part right quite yet. I'll remove it but I think we need to get some practical experience with what such long view names look like in the UI of common metrics backends.